config/locales/en/helpers.yml

```diff
@@ -64,6 +70,8 @@ en:
         # default key
         known_date: When were you given the order?
         # alternative keys defined with `i18_attribute`
+        dismissal: When were you given the dismissal?
+        service_detention: When were you given the dismissal?
```

Comment:
This needs to be `When were you given the detention?` according to the spreadsheet.